examples/client/check_ssl/check_ssl.go

```diff
@@ -1,25 +1,25 @@
 package main
 
 import (
-	"log"
-
 	"github.com/bitcoin-sv/go-paymail"
+	"github.com/bitcoin-sv/go-paymail/logging"
 )
 
 func main() {
+	logger := logging.GetDefaultLogger()
```

Comment:
It is only example,
I'm not sure if we should use logger here.

Reply:
Done